client/src/main/java/org/eclipse/hono/client/MessageSender.java

```diff
@@ -27,70 +27,30 @@
  * A client for publishing messages to Hono.
  *
  */
-public interface MessageSender {
+public interface MessageSender extends CreditBasedSender{
```

Comment:
missing whitespace before opening braces

Reply:
I already run the checkstyle plugin. It didn't report any error. I will change this.